internal/options/options.go

```diff
@@ -9,16 +9,16 @@ import (
 )
 
 type CollectorRunOptions struct {
-	MetricResolution      time.Duration
+	PushInterval          time.Duration
```

Comment:
Let's remove this from here. Flags will be deprecated in favor of config going forward. All new additions will be config only.

Reply:
Should I keep `MetricResolution` ?